ext/README.md

```diff
@@ -3,6 +3,38 @@
 CEL extensions are a related set of constants, functions, macros, or other
 features which may not be covered by the core CEL spec.
 
+## Encoders
+
+Encoding utilies for marshalling data into standardized representations.
+
+### Base64.Decode
+
+Decodes bytes or a base64-encoded string to a sequence of bytes.
+
+This function will return an error if the UTF-8 string input is not
+base64-encoded.
+
+    base64.decode(<bytes>)  -> <bytes>
```

Comment:
Specify that the bytes are interpreted as a utf8-encoded string.

Reply:
Actually, the `bytes` are just treated as a raw `byte` sequence with no assumptions of utf-8 encoding.

Reply:
Then how do you interpret the sequence of bytes as characters without assuming the character encoding?  What if I give it some base64-encoded text, using utf16 as the text-to-binary encoding? If the Go library decodes a sequence of bytes, it must do it by assuming a text encoding. That's the assumption that must be surfaced.